specs/RuntimeSpec/Fees/Fees.md

```diff
@@ -15,10 +15,11 @@ Every [Fee](/GenesisConfig/RuntimeFeeConfig/Fee.md) consists of 3 values measure
     - `send_not_sir` is used when `current_account_id != receiver_id`
 - `execution` - the gas burned when the action is being executed on the receiver's account.
 
-Burning gas is different from charging gas:
-- Burnt gas is not refunded.
-- Charged gas can potentially be refunded in case the execution stopped earlier and the remaining
-actions are not going to be executed. So the charged gas for the remaining actions can be refunded.
+We track gas which have to be deducted in two values:
+- Burnt gas - irreversible amount of gas witch was spent on computations.
+- Used gas - includes burnt gas and gas attached to the new `ActionReceipt`s created during the method execution. 
+  
+Initially system charges all used gas from the account. But in case of failure, created `ActionReceipt`s will not be sent and difference between used gas and burnt gas will be refunded.
```

Comment:
```suggestion
Initially runtime charges all used gas from the account. But in case of failure, created `ActionReceipt`s will not be sent and difference between used gas and burnt gas will be refunded.
```